defaults/main.yml

```diff
@@ -104,14 +104,20 @@ roles: null
 all_rw: null
 vshard_group: null
 weight: null
-edit_topology_timeout: 60
+
+twophase_netbox_call_timeout: null
+twophase_upload_config_timeout: null
+twophase_apply_config_timeout: null
+
+edit_topology_timeout: null  # DEPRECATED
```

Comment:
I see no warnings `Option "edit_topology_timeout" is deprecated and will be removed in future release, use "edit_topology_healthy_timeout" instead`.

Reply:
It's a good idea to make warnings for deprecated variables!

Reply:
It's a good idea to make warnings for deprecated variables!